precheck/lib/precheck/rule_processor.rb

```diff
@@ -175,10 +176,34 @@ def self.generate_text_items_to_check(app: nil, app_version: nil)
                                                 item_name: :app_subtitle,
                                     friendly_name_postfix: "app name subtitle",
                                               is_optional: true)
+
+      should_include_iap = Precheck.config[:include_in_app_purchases]
+      if should_include_iap
+        UI.message "Reading in-app purchases. If you have a lot, this might take a while"
+        UI.message "You can disable IAP checking by adding the include_in_app_purchases flag"
```

Comment:
Should this say "...checking by setting the `include_in_app_purchases` flag to `false`"